From 535de850f820faac9443a60edca6b30221cc25df Mon Sep 17 00:00:00 2001
From: Michal Simek <monstr@monstr.eu>
Date: Mon, 8 Oct 2012 12:51:52 +0200
Subject: [PATCH] block: Makefile: Remove duplicate reference for sysace

Do not define compilation rule for sysace because
it is already defined.

Signed-off-by: Michal Simek <monstr@monstr.eu>
---
 drivers/block/Makefile |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/drivers/block/Makefile b/drivers/block/Makefile
index 3814ded..b20c87f 100644
--- a/drivers/block/Makefile
+++ b/drivers/block/Makefile
@@ -34,7 +34,6 @@ obj-$(CONFIG_VIRTIO_BLK)	+= virtio_blk.o
 obj-$(CONFIG_VIODASD)		+= viodasd.o
 obj-$(CONFIG_BLK_DEV_SX8)	+= sx8.o
 obj-$(CONFIG_BLK_DEV_UB)	+= ub.o
-obj-$(CONFIG_XILINX_SYSACE)	+= xsysace.o
 obj-$(CONFIG_XILINX_SYSACE_OLD)	+= xilinx_sysace/
 obj-$(CONFIG_BLK_DEV_HD)	+= hd.o
 
-- 
1.7.0.4

